README.md

```diff
@@ -21,43 +21,35 @@ environment. It acts like an ether, an omnipresent, persistent layer
 for data. This way, application containers access their data
 regardless on which host the data nodes run.
 
-[Crate: Put your data to work. Simply.](https://crate.io/)
+[Crate](https://crate.io/)
 
-## Building Images
+![logo](https://raw.githubusercontent.com/docker-library/docs/2517900006ae5f4c03c1d43235930c59f4614394/crate/logo.png)
 
-Crate derives from
-[alpine:latest](https://hub.docker.com/_/alpine/). We build this
-nightly updated image from the `Dockerfile` on GitHub:
-
-    docker build -t="crate/crate" github.com/crate/docker-crate
-
-To build your own image, replace the image name after the `-t` option
-with your own name.
-
-Alternativly, an [Official Crate
-Image](https://hub.docker.com/_/crate/) with the name `crate` is
-available in the Docker Hub. That image is based on a stable major
-version of Crate, and is tested and scanned by Docker, Inc.
-
-## How to use this image
```

Comment:
Having a section explaining how to build the actual image is a good thing -- at least for `crate/crate`. It makes the build process more transparent and reproducible for users. We should at least keep the pointer to the GH repo and add some explanation what's the difference to the `crate` image, otherwise this is very confusing.